recipe/meta.yaml

```diff
@@ -1,21 +1,20 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "3.3.4" %}
+{% set version = "3.4.2" %}
 
 package:
   name: matplotlib-suite
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: ce7cebd3df11a032fff9b3a32029f49f0fc7c322f556f26ea9ef812d6e43cb21
+  sha256: e5960bcb964ee77d37752f7a1db4568d85e99d8857fbf20289dda2edfa711675
   patches:                    # [osx]
     - fix_osx_1011_dep.patch  # [osx]
 
 build:
   number: 0
-  skip: true  # [py<36]
```

Comment:
I would suggest trying to update this to `skip: true  # [py<37]`. That way it applies to both `matplotlib-base` & `matplotlib`. 

If that does not work it might be due to the empty `build:` key for the `matplotlib` output. I am not sure that should be there. 🤔 

However, if that is not able to work, I am fine with these changes as is.